utils/pom.xml

```diff
@@ -61,7 +61,7 @@
               io.atomix.utils.*
             </Export-Package>
             <Import-Package>
-              !sun.nio.ch,!sun.misc,*
+              sun.nio.ch;resolution:=optional,sun.misc;resolution:=optional,*
```

Comment:
@kuujo I think this makes more sense as sun.* should only be required if using memory / mapped storage level, right?